client/state/action-types.js

```diff
@@ -6,6 +6,8 @@
  */
 
 export const FAIL_PUBLICIZE_CONNECTIONS_REQUEST = 'FAIL_PUBLICIZE_CONNECTIONS_REQUEST';
+export const FETCH_SITE_PLANS = 'FETCH_SITE_PLANS';
+export const FETCH_SITE_PLANS_COMPLETED = 'FETCH_SITE_PLANS_COMPLETED';
```

Comment:
>which makes it difficult to group them by the areas they correspond to

By area, do you mean the subtrees of states that use the actions? If so, part of the goal with moving action types to a single file in the root directory is to try to move away from this line of thinking. Rather, we should promote the idea no one particular area _owns_ any action type and that any subtrees can dispatch or subscribe to any action type. 

Alphabetizing is pretty insignificant to me, but I think there's a few minor benefits: ease of lookup when name is known, no confusion around where a new action type should be inserted, and ease of reference by intent (fetching, receiving, etc.).

>Since not all API calls are necessarily 'fetching' data

I personally don't think we should feel forced to using few prefixes, and rather choose whichever is most appropriate for the intent of the action. That said, `REQUEST` and `FETCH` are a bit synonymous - maybe in these cases we should choose one. In the example you provided, you could probably choose one that's even more descriptive of the action, such as `TRIGGER_START_EXPORT` ?